### ISSUE TITLE
[GPU] Workflow failures from missing DQM_HcalReconstruction_v6

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#44046]
Several workflows {12434,12450}.{402,403,404,412} fail in GPU IB tests in CMSSW_14_1_GPU_X_2024-02-20-2300 along
```
DIGI:pdigi_valid,L1,DIGI2RAW,HLT:@relval2023,ENDJOB
We have determined that this is simulation (if not, rerun cmsDriver.py with --data)
with DB:
entry filelist:step1_dasquery.log
found files:  ['/store/relval/CMSSW_13_0_10/RelValTTbar_14TeV/GEN-SIM/130X_mcRun3_2023_realistic_withEarly2023BS_v1_2023-v1/2590000/4a9c4099-1812-4afd-9c94-6f9409595929.root', '/store/relval/CMSSW_13_0_10/RelValTTbar_14TeV/GEN-SIM/130X_mcRun3_2023_realistic_withEarly2023BS_v1_2023-v1/2590000/99db1b20-ec34-4bff-84df-dfffcbdfb184.root', '/store/relval/CMSSW_13_0_10/RelValTTbar_14TeV/GEN-SIM/130X_mcRun3_2023_realistic_withEarly2023BS_v1_2023-v1/2590000/c388e800-ddaa-408d-a2ec-b40a9b8c7a08.root']
Step: DIGI Spec: ['pdigi_valid']
Step: L1 Spec: 
Step: DIGI2RAW Spec: 
Step: HLT Spec: ['@relval2023']
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 40, in <module>
    run()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 16, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/Configuration/Applications/python/ConfigBuilder.py", line 2310, in prepare
    self.addStandardSequences()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/Configuration/Applications/python/ConfigBuilder.py", line 850, in addStandardSequences
    getattr(self,"prepare_"+stepName)(stepSpec = '+'.join(stepSpec))
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/Configuration/Applications/python/ConfigBuilder.py", line 1670, in prepare_HLT
    self.loadAndRemember('HLTrigger/Configuration/HLT_%s_cff' % stepSpec)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/Configuration/Applications/python/ConfigBuilder.py", line 376, in loadAndRemember
    self.process.load(includeFile)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/FWCore/ParameterSet/python/Config.py", line 761, in load
    module = __import__(moduleName)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/HLTrigger/Configuration/python/HLT_2023v12_cff.py", line 88697, in <module>
    fragment = customizeHLTforCMSSW(fragment,"2023v12")
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/HLTrigger/Configuration/python/customizeHLTforCMSSW.py", line 252, in customizeHLTforCMSSW
    (alpaka & run3_common).makeProcessModifier(customizeHLTforAlpaka).apply(process)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/FWCore/ParameterSet/python/Config.py", line 1980, in apply
    self.__func(process)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/HLTrigger/Configuration/python/customizeHLTforAlpaka.py", line 911, in customizeHLTforAlpaka
    process = customizeHLTforAlpakaParticleFlowClustering(process)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/HLTrigger/Configuration/python/customizeHLTforAlpaka.py", line 250, in customizeHLTforAlpakaParticleFlowClustering
    dqmHcalPath= getattr(process, dqmHcalRecoPathName)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02825/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_GPU_X_2024-02-20-2300/src/FWCore/ParameterSet/python/Config.py", line 1656, in __getattribute__
    return getattr(self.__process, name)
AttributeError: 'Process' object has no attribute 'DQM_HcalReconstruction_v6'
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/el8_amd64_gcc12/CMSSW_14_1_GPU_X_2024-02-20-2300/pyRelValMatrixLogs/run/12434.402_TTbar_14TeV+2023_Patatrack_PixelOnlyAlpaka/step2_TTbar_14TeV+2023_Patatrack_PixelOnlyAlpaka.log#/

**Comment by makortel** [comment]
assign hlt, heterogeneous

**Comment by makortel** [comment]
https://github.com/cms-sw/cmssw/pull/43999 looks likely cause

**Comment by mmusich** [comment]
+hlt
   * fixed in master by #44048 and backported as part of https://github.com/cms-sw/cmssw/pull/44000 in 14.0.x

**Comment by makortel** [comment]
+heterogeneous
